pkg/util/sys/device.go

```diff
@@ -224,9 +237,15 @@ func GetDiskUUID(device string, executor exec.Executor) (string, error) {
 		return "sgdiskNotFound", nil
 	}
 
+	devicePath := strings.Split(device, "/")
```

Comment:
why is this needed?

Reply:
As we are mounting the block mode PVs on /mnt we use the entire path,
e.g., if the device path is `/mnt/example-pvc` then its taken completely else if its just `vdb` then the following is used
 ```devicePath := strings.Split(device, "/")
	if len(devicePath) == 1 {
		device = fmt.Sprintf("/dev/%s", device)
	}
```